File: python/734A.py
input()

s = input()
count_a = count_d = 0
for i in s:
    if i == 'A':
        count_a+=1
    else:
        count_d+=1
if count_a > count_d:
    print('Anton')
elif count_a < count_d:
    print('Danik')
else:
    print('Friendship')

File: golang/round379/734A.go
package main

import "fmt"

func main()  {
	var n int
	var s string
	fmt.Scan(&n)
	fmt.Scan(&s)
	d := 0
	a := 0
	for _, c := range s {
		if c == 'A' {
			a += 1
		} else {
			d += 1
		}
	}
	if d > a {
		fmt.Println("Danik")
	} else if d == a {
		fmt.Println("Friendship")
	} else {
		fmt.Println("Anton")
	}
}
